website/.eslintrc.json

```diff
@@ -6,6 +6,19 @@
     "next/core-web-vitals"
   ],
   "rules": {
-    "sort-imports": "warn"
-  }
+    "simple-import-sort/imports": "error",
+    "simple-import-sort/exports": "error",
+    "@typescript-eslint/no-unused-vars": "off",
+    "unused-imports/no-unused-imports": "error",
+    "unused-imports/no-unused-vars": [
+      "warn",
+      {
+        "vars": "all",
+        "varsIgnorePattern": "^_",
+        "args": "after-used",
+        "argsIgnorePattern": "^_"
+      }
+    ]
+  },
```

Comment:
I know that customizing eslint is always very opinionated... imo there is no need for these rules.

I will have to wait for opinions of other team member...

Reply:
I've changed it to keep the less rule as possible to make it work:
```
"unused-imports/no-unused-imports": "warn",  //mandatory to automatically remove unused import  
"simple-import-sort/imports": "warn", //mandatory to automatically sort import  
"simple-import-sort/exports": "warn"  //mandatory to automatically sort export
```

Reply:
I think these three small additions look good.